engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/hv/EmailValidatorTest.java

```diff
@@ -54,36 +55,67 @@ public void testNullAndEmptyString() throws Exception {
 
 	@Test
 	public void testValidEmail() throws Exception {
-		isValidEmail( "emmanuel@hibernate.org" );
-		isValidEmail( "emmanuel@hibernate" );
-		isValidEmail( "emma-n_uel@hibernate" );
-		isValidEmail( "emma+nuel@hibernate.org" );
-		isValidEmail( "emma=nuel@hibernate.org" );
-		isValidEmail( "emmanuel@[123.12.2.11]" );
-		isValidEmail( "*@example.net" );
-		isValidEmail( "fred&barny@example.com" );
-		isValidEmail( "---@example.com" );
-		isValidEmail( "foo-bar@example.net" );
-		isValidEmail( "mailbox.sub1.sub2@this-domain" );
+		Arrays.asList(
+				"emmanuel@hibernate.org",
+				"emmanuel@hibernate",
+				"emma-n_uel@hibernate",
+				"emma+nuel@hibernate.org",
+				"emma=nuel@hibernate.org",
+				"emmanuel@[123.12.2.11]",
+				"*@example.net",
+				"fred&barny@example.com",
+				"---@example.com",
+				"foo-bar@example.net",
+				"mailbox.sub1.sub2@this-domain",
+				"prettyandsimple@example.com",
+				"very.common@example.com",
+				"disposable.style.email.with+symbol@example.com",
+				"other.email-with-dash@example.com",
+				"x@example.com",
+				"\"much.more unusual\"@example.com",
+				"\"very.unusual.@.unusual.com\"@example.com",
+				"\"very.(),:;<>[]\\\".VERY.\\\"very@\\\\ \\\"very\\\".unusual\"@strange.example.com",
+				"\"some \".\" strange \".\" part*:; \"@strange.example.com",
+				"example-indeed@strange-example.com",
+				"admin@mailserver1",
+				"#!$%&'*+-/=?^_`{}|~@example.org",
+				"\"()<>[]:,;@\\\"!#$%&'-/=?^_`{}| ~.a\"@example.org",
+				"\" \"@example.org",
+				"example@localhost",
+				"example@s.solutions",
+				"user@localserver",
+				"user@tt",
+				"user@[IPv6:2001:DB8::1]" )
+				.forEach( this::isValidEmail );
```

Comment:
I'm not convinced by this change. I agree it looks better but when you have one address failing, it's far harder to find it because you can't count on your IDE leading you to the right line.

Would you mind going back to the previous pattern?